core/src/main/java/org/bitcoinj/base/utils/StreamUtils.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright by the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bitcoinj.base.utils;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collector;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Stream Utilities. bitcoinj is moving towards functional-style programming, immutable data structures, and
```

Comment:
Nitpick: I think when the word "bitcoinj" is used as first word of a sentence it should read "Bitcoinj", like any other word.

Reply:
Fixed and force-pushed.